docs/ABOUT.md

```diff
@@ -1,5 +1,7 @@
-C# is a modern, object oriented language developed and maintained by Microsoft. The learning curve is small, reduced even if the developer has experience with other 'C-Style' languages.
+C# is a multiparadigm language with lots of features. While at its core it's still an object oriented language that's modeled after Java it supports first class functions, SQL-like syntax to access containers, automatic getters and setters, pattern matching, direct memory access in an otherwise garbage collected environment, statically typed, dynamically typed and (to a certain degree) type inferred programming and many other features in a constantly growing feature set.
```

Comment:
I think multiparadigm should be multi-paradigm.

Reply:
Although it was indeed modelled after Java, it has long since surpassed that language. Hence, I think mentioning that it is a 'C-Style language' is a better wording.

Reply:
Should we mention dynamic typing? I hardly see anyone using it, so is this really a big advantage?